python/ycm/tests/postcomplete_tests.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python
+#
+# Copyright (C) 2013  Google Inc.
```

Comment:
This file is your copyright, not Google's. :)

You could also, if you wish, make it "YouCompleteMe Contributors".

Also the year is wrong.